src/webgl/context-state.js

```diff
@@ -535,26 +536,15 @@ export function setParameter(gl, key, value) {
 export function withState(gl, params, func) {
   // assertWebGLContext(gl);
   const state = getState(gl);
-
```

Comment:
Remove framebuffer settings from withState as framebuffer state currently is not tracked by our state manager. The ideal solution would be having GL state manager track it but remove *for now* for consistency.